### DISCUSSION TITLE
wrong colors for html

### DISCUSSION THREAD

**Discussion by slavafyi** [projekt0n/github-nvim-theme#217] [Q&A] [answered]
Hey there! I like the github light default theme, but some of the colors don't match. This is what it should look like:

```liquid
<ul hidden>
  <li id="a11y-new-window-message"></li>
</ul>

<script>
  window.routes = {
    cart_add_url: 'add',
    cart_change_url: 'change',
    cart_update_url: 'update',
  }
</script>
```

I'm trying to override the colors, but it doesn't seem to work

```lua
overrides = function(c)
  return {
    htmlArg = { fg = c.blue },
    htmlSpecialTagName = { fg = c.green },
  }
end
```

<img width="410" alt="Screenshot 2022-11-30 at 21 50 46" src="https://user-images.githubusercontent.com/28267670/204883603-14ba9160-12d1-40c8-a4fb-7e8232d75114.png">
<img width="446" alt="Screenshot 2022-11-30 at 21 50 36" src="https://user-images.githubusercontent.com/28267670/204883623-cff15e54-9ba9-473a-8670-769735e07d43.png">

Maybe I'm doing something wrong? Any ideas
I use neovim 8.0, nvim-treesitier

**Comment by ful1e5** [comment]
Only a few highlights have been defined for Html in this theme, but the highlights aren't always the same due to constant changes in the treesitter. But still, you can override the highlights not already defined in this theme by enabling `dev` mode.

```lua
require('github-theme').setup({
  -- ...
  overrides = function(_)
    return {
    htmlArg = { fg = c.blue },
    htmlSpecialTagName = { fg = c.green },
    }
  end,
  dev = true,
})

```

**Reply by slavafyi** [reply]
yes thanks, it seems to work. But still I can't override some things, like operators or semicolon.
Is this a treesitter problem or is this something that can be fixed with the theme?

<img width="517" alt="Screenshot 2022-12-01 at 15 36 54" src="https://user-images.githubusercontent.com/28267670/205055460-467956ba-9cea-44a9-bbdb-5fdd23d6aa16.png">
<img width="612" alt="Screenshot 2022-12-01 at 15 41 12" src="https://user-images.githubusercontent.com/28267670/205055474-0c108d70-80da-4d7c-89da-771d4ce5d892.png">

**Comment by tmillr** [comment, marked as answer]
@slavamak it has been some months since this was originally posted, but just letting you know that alot of these issues have been fixed now and all you should need to do is update the plugin

**Reply by slavafyi** [reply]
yeah, it seems to work good, thanks!
